src/main/java/graphql/schema/GraphQLFieldDefinition.java

```diff
@@ -185,11 +163,7 @@ public Builder deprecate(String deprecationReason) {
 
         public GraphQLFieldDefinition build() {
             if (dataFetcher == null) {
-                if (isField) {
-                    dataFetcher = new FieldDataFetcher(name);
-                } else {
-                    dataFetcher = new PropertyDataFetcher(name);
-                }
+                dataFetcher = new PropertyDataFetcher(name);
```

Comment:
@markphilpot agreed that verbose is not good. but i'd prefer verbose to ambiguous. the ambiguity i think in using the propertyfetcher is the assumption that field is available with getters when it could very well be a map. or worse no getters or setters :laughing:. improbable but possible.